### PR TITLE
feat: change tag of a block to title or paragraph

### PR DESCRIPTION
**Pull request by ReneCode**
solves: #9

change tag of existing block
triggered if block is '/'
added switching also to paragraph

### CODE REVIEW

File: src/components/EditableBlock.vue [hunk 1]
```diff
@@ -100,6 +120,11 @@
         @clicked-italic="surroundWith('em')"
       />
     </div>
+    <ChangeType
+      v-if="showChangeType"
+      @clicked-title="changeBlockTag('h2')"
+      @clicked-paragraph="changeBlockTag('p')"
+    ></ChangeType>
```
Comment (Vannsl) [inline src/components/EditableBlock.vue:127]: [nit] human linter: for the sake of consistency I suggest to use self-closing tags:

```
<ChangeType
      v-if="showChangeType"
      @clicked-title="changeBlockTag('h2')"
      @clicked-paragraph="changeBlockTag('p')"
 />
```

File: src/components/ChangeType.vue [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      color="primaryFlat"
+      title="Paragraph"
+      @clicked="emit('clicked-paragraph')"
+    >
+      Paragraph
+    </BaseButton>
+  </div>
+</template>
+
+<script setup>
```
Comment (Vannsl) [inline src/components/ChangeType.vue:36]: [nit] for sake of consistency, I would move the `script` block before the `template` block. This is not as shown as in the examples of Vue applications but a preferred method of bigger applications (and me :D )

**Comment by Vannsl** [comment]
nice! I see some follow ups, such as Subcategories within the drop down (e.g. "Text Elements": Title + Paragraph, "Media": Image, "External": YouTube Video (yeah I dream big :D) and making the popover accessible by sending the focus inside when it's opened. But in general it works! :)

**Comment by ReneCode** [comment]
> nice! I see some follow ups, such as Subcategories ...

yes a lot of improvements possible. Let's make them step by step.

**Comment by ReneCode** [comment]
Should the merged branch be deleted afterwards ?

**Pull request merged**